.travis.yml

```diff
@@ -3,13 +3,14 @@ node_js:
   - 8
   - 10
   - stable
-sudo: false
 
 git:
   submodules: false
 
 install:
   - yarn --frozen-lockfile
+  - yarn pretest
```

Comment:
Maybe comment why this is necessary. (Because the next line needs the submodules synced.)

Or I wonder if there's a specific Travis thing for including submodules in the clone.

Reply:
Per df2e6189b, the built-in submodule support was previously insufficient for our use cases as wpt had its own submodules. wpt no longer seems to have any submodules. I'm going to experiment with using the default support a bit.

Reply:
Looks like the built-in support works.